src/Illuminate/Notifications/SendQueuedNotifications.php

```diff
@@ -25,14 +25,23 @@ class SendQueuedNotifications implements ShouldQueue
     protected $notification;
 
     /**
+     * All of the channels to send the notification too.
+     *
+     * @var array
+     */
+    protected $channels = null;
+
+    /**
      * Create a new job instance.
      *
      * @param  \Illuminate\Support\Collection  $notifiables
      * @param  \Illuminate\Notifications\Notification  $notification
+     * @param  array  $channels
      * @return void
      */
-    public function __construct($notifiables, $notification)
+    public function __construct($notifiables, $notification, array $channels = null)
     {
+        $this->channels = $channels;
```

Comment:
Minor: SHould you put this line after the other two, since it was the last item in the method signature?